src/CheckoutGateway.php

```diff
@@ -59,6 +60,8 @@ public function getName()
 
     public function purchase(array $parameters = []): RequestInterface
     {
+        $request = $this->createRequest(RegisterWebhookRequest::class, $parameters);
+        $request->send();
```

Comment:
I guess we are throwing away failures here. Not sure what to do about that though, I guess nothing :(

$response = $request->send();
if ($response->isSuccessful()) { ...???

Reply:
The request will throw an exception already if there's a failure getting/creating the webhook.